ci/lib.sh

```diff
@@ -104,21 +104,3 @@ export OS
 # RELEASE_PATH is the destination directory for the release from the root.
 # Defaults to release
 RELEASE_PATH="${RELEASE_PATH-release}"
-
-# VS Code bundles some modules into an asar which is an archive format that
-# works like tar. It then seems to get unpacked into node_modules.asar.
-#
-# I don't know why they do this but all the dependencies they bundle already
-# exist in node_modules so just symlink it. We have to do this since not only VS
-# Code itself but also extensions will look specifically in this directory for
-# files (like the ripgrep binary or the oniguruma wasm).
-symlink_asar() {
```

Comment:
We likely need to keep these symlinks. When I built a release there was no `vendor/modules/code-oss-dev/node_modules.asar` directory which some extensions rely on.

To reproduce, install the Bracket Pair Colorizer 2 extension. You will see an error in the browser console and the brackets will not be colored.

I think VS Code does something where they duplicate some modules into this directory so maybe eventually use their build process instead although it seems like a waste of space. :man_astronaut: